packages/react/src/hooks/useIsConnected.ts

```diff
@@ -2,6 +2,21 @@ import { useNamedQuery } from '../core';
 import { useFuel } from '../providers';
 import { QUERY_KEYS } from '../utils';
 
+/**
+ * `useIsConnected` is a React Hook to check the connection status with the connector.
+ * The connection status is fetched using the connected connector's `isConnected` method.
+ *
+ * @see {@link https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/src/connectors/fuel-connector.ts | fuel-connector.ts on `FuelLabs/fuel-ts`}
+ *
+ * @returns {object} An object containing:
+ * - `data`: A boolean value indicating the connector is connected.
+ * - Additional properties from `useNamedQuery`.
+ *
+ * @example To check if a connection is established:
+ * ```ts
+ * const { isConnected } = useIsConnected();
+ * ```
+ */
```

Comment:
3 changes here,

1- doc link should send the user to this hook link inside of "Hooks Reference" section that will be created in doc
2- the value returned is the `isConnected` instead of `data`
3- the "Additional properties from `useNamedQuery`" should be changed to "Properties of `@tanstack/react-query` , `useQuery` method" and include a link to it


apply this change to other hooks also if needed